features/environment.py

```diff
@@ -16,7 +16,11 @@ def before_all(_):
     logging.captureWarnings(True)
 
 
-def before_scenario(context, _):
+def before_scenario(context, scenario):
```

Comment:
Might be worth mentioning in the README about adding the `@skip` tag to the scenario which tests the 3.5 mil sample if running locally? Either that, or maybe we could skip over this test if running locally by tagging the scenario with `skip-local-dev` and then add that tag to the Makefile to ignore when these tests are run locally?

Reply:
need to decide